src/Dapr.Client/InvokeHttpClient.cs

```diff
@@ -66,7 +66,7 @@ public InvokeHttpClient(HttpClient client, JsonSerializerOptions serializerOptio
                 throw new ArgumentNullException("The value cannot be null", nameof(data));
             }
 
-            var response = await this.MakeInvokeHttpRequest(serviceName, methodName, JsonSerializer.Serialize(data), cancellationToken).ConfigureAwait(false);
```

Comment:
There's another case that's wrong on L143.

Possible to add a test?

Reply:
Sure! Let me add it